core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/ContainerProxy.scala

```diff
@@ -572,6 +572,7 @@ class ContainerProxy(
           "namespace" -> job.msg.user.namespace.name.toJson,
           "action_name" -> job.msg.action.qualifiedNameWithLeadingSlash.toJson,
           "activation_id" -> job.msg.activationId.toString.toJson,
+          "transaction_id" -> tid.id.toJson,
```

Comment:
Should this be `job.msg.transid.id` instead? In principle that should be the same as `tid`.